packages/terra-form/docs/Input.md

```diff
@@ -0,0 +1,16 @@
+# Input
+
+Generic input which represents an HTML input element directly.
+
+## Usage
+
+```jsx
+import React from 'react';
+import { Input } from 'terra-form';
+
+<Input
+  required={false}
```

Comment:
In our early meetings I thought we had discussed not using html attributes as prop names, but I could be remembering that conversation incorrectly, it was a while back.